e2e/app.step1.e2e.ts

```diff
@@ -14,41 +14,63 @@ describe('angular2-multicomponent-form App Step1 Tests: ', function() {
     done();
   });
 
+  it('should change input Area by changing input Region', (done) => {
+    page.setInputFieldValue('input_region', 'East of England');
```

Comment:
Assertions are done before the change and after the change to ensure that something has actually changed. Right you you are just checking a value.